README.md

```diff
@@ -25,7 +26,6 @@ CompStore web store frontend
 
 ## Other used sources
 
--   Source Concept: https://www.euro.com.pl/
```

Comment:
Rozwiązałem konflikty.
Tak myślę, że może by usunąć źródło wzorowania się? Wg mnie bardziej liczy się efekt, a nie czy jest do czegoś podobne, a jakieś o wielkości promila zawsze jest, że komuś się to nie spodoba

Reply:
Z mojej strony approve. Jeśli się zgadzasz to możesz zmergować.

Reply:
OK, możemy usunąć żródło wzorcowania się.